docs/source/pages/documentation.md

```diff
@@ -1,351 +1,631 @@
 
-### transfer_verbosity
+Similarly, if the test session was on different session numbers for different sessions, we could use `-ses @*@condition-test@*@` (as the `-ses` argument above) to selectively transfer test sessions only.
 
-When set to vv, the console and log output will become very verbose and report all defaults. When
-v (default) these will report each file that is transferred and a small number of significant events.
+[TODO] all links are probably broken.
 
-### show_transfer_progress
+## Data Transfer Options
 
-When true, real-time transfer statistics will be reported and logged.
+Behind the scenes, DataShuttle uses the software [Rclone](https://rclone.org/) to transfer data. A number of Rclone options are exposed in DataShuttle to allow for more flexible data transfer. These are set in the *configurations* during project initialisation and can be edited with the command `update-config`.
 
-### All sub_names, ses_names and data_type keywords
+#### Overwriting existing files
 
-For each argument, the subject, session or datatype to transfer can be specified directly, e.g.
-`project.upload_data(sub_names="sub-001", ses_names=["ses-001", "ses-002]", data_type="behav" )`
+The most important optional configuration, `overwrite_old_files` determines whether folders and files are overwritten during transfer. By default, DataShuttle does not overwrite during data transfer.
 
-However, a number of keyword arguments can be used to specify more general rules for transfer:
+For example, if the file `sub-001_ses-001_measure-trajectories.csv` has been transferred from *local* to the *remote* repository, it will never be over-written in the *remote* repository. If the file is edited locally, such that the timestamp on the file is more recent in *local* compared to *remote*, the *remote* version will still not be overwritten.
 
-*sub_names*
+To change this behaviour, the configuration `overwrite_old_files` can be set to `True`. In this case, files in which the timestamp of the target directory (e.g. *central* when performing `upload`) will be overwritten if their timestamp is older than the corresponding file in the source directory (e.g. *local* when performing `upload`.
+)
 
-<u> all </u>: all subjects and any non-subject files or folders at the top level (e.g. under rawdata)
-will be transferred <br>
+#### Additional Transfer Configurations
 
-<u> all_sub </u>: all subjects, but not any non-subject files or folders at the top level will be transferred  <br>
+`transfer_verbosity` : set to `"vv"` for a extensive detail on the transfer operation. Set to `"v"` to see simply each file that is transferred as well as significant events that occur during transfer.
 
-<u> all_non_sub </u>: Only non-subject folders (or files) will be transferred from the top level <br>
+`show_transfer_progress` : When `True`, real-time transfer statistics will be reported and logged.
 
-*ses_names*
+#### Specifying transfers with keyword arguments [TODO better name]
```

Comment:
I found this section quite confusing. The commands and their effects are not really intuitive. We should plan to rethink these funcitonalities at some point. We could discuss this during the team meeting presentation.